src/diagnostics.rs

```diff
@@ -522,6 +522,22 @@ fn print_snippet(
   Ok(())
 }
 
+fn to_file_path_if_not_wasm(specifier: &ModuleSpecifier) -> Option<PathBuf> {
+  #[cfg(target_arch = "wasm32")]
+  {
+    None
+  }
+  #[cfg(not(target_arch = "wasm32"))]
+  {
+    if specifier.scheme() == "file" {
```

Comment:
Btw, when using `to_file_path` we need to remember to do this first (I'm not sure why they didn't just build this into to_file_path)